code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -375,39 +387,6 @@ GLOBAL_LIST_INIT(titanium_recipes, list(
 	amount = 50
 
 
-/*
- * Plastitanium
- */
-/obj/item/stack/sheet/mineral/plastitanium
-	name = "plastitanium"
-	icon_state = "sheet-plastitanium"
-	item_state = "sheet-plastitanium"
-	singular_name = "plastitanium sheet"
-	force = 5
-	throwforce = 5
-	w_class = WEIGHT_CLASS_NORMAL
-	throw_speed = 1
-	throw_range = 3
-	sheettype = "plastitanium"
-	merge_type = /obj/item/stack/sheet/mineral/plastitanium
-	materials = list(MAT_TITANIUM=2000, MAT_PLASMA=2000)
-	point_value = 45
-
-GLOBAL_LIST_INIT(plastitanium_recipes, list(
-	new /datum/stack_recipe("plas-titanium tile", /obj/item/stack/tile/mineral/plastitanium, 1, 4, 20),
-	new /datum/stack_recipe("Kidan Warrior Statue", /obj/structure/statue/plastitanium/kidanstatue, 5, time = 2.5 SECONDS, one_per_turf = TRUE, on_floor = TRUE),
-	))
-
-/obj/item/stack/sheet/mineral/plastitanium/New(loc, amount=null)
-	recipes = GLOB.plastitanium_recipes
-	..()
-
-/obj/item/stack/sheet/mineral/enruranium
-	name = "enriched uranium"
-	icon_state = "sheet-enruranium"
-	origin_tech = "materials=6"
-	materials = list(MAT_URANIUM=3000)
-
```

Comment:
You werent meant to delete this.

Also please actually test your PRs.

Reply:
Plastitanium sheets definitely do have a sprite. You can get them from the ORM too.
![image](https://user-images.githubusercontent.com/91113370/206934790-706dada0-05a7-4ae6-9875-458e892a24dc.png)

Reply:
Yeah, what happened is that I made a typo when coding plastinium/fifty and then got confused because the typod thing didn't exist